### DISCUSSION TITLE
Trojan:Win32/Zpevdo.B being detected by Microsoft Security/Windows Defender in 3.6.2

### DISCUSSION THREAD

**Discussion by KTac** [SubtitleEdit/subtitleedit#5283] [General]
Out of an abundance of caution, I would like to know if this is a false positive or if there is cause for alarm.

**Comment by OmrSi** [comment]
It's a false positive: https://github.com/SubtitleEdit/subtitleedit/issues/5280

**Reply by KTac** [reply]
@OmrSi Thanks!
